docs/2.8.0/docs/daml/reference/templates.rst

```diff
@@ -92,7 +92,58 @@ Template-local Definitions
    - interface methods applied to ``this``
 
    in a template-local let definition will result in an error generated by the infinite loop.
- 
+
+.. warning::
```

Comment:
Can we remove the 'warning' label about the infinite loop discussion?  Otherwise, there are two warnings in a row and that means people will miss the importance of the second warning.  I also think the infinite loop can be used as justification for deprecating the feature.

Reply:
done, I moved the infinite loop discussion into the deprecation warning as justification, thanks for the suggestion. Could you have another look at that please?